docs/styles/classes.md

```diff
@@ -1,10 +1,21 @@
 ### CSS Classes
 
-Sometimes it may be preferable to use CSS classes rather than inline styles.  You can use the `className` and `overlayClassName` props to specify a given CSS class for each of those.
-You can override the default class that is added to `document.body` when the modal is open by defining a property `bodyOpenClassName`.
+Sometimes it may be preferable to use CSS classes rather than inline styles.
 
-It's required that `bodyOpenClassName` must be `constant string`, otherwise we would end up with a complex system to manage which class name
-should appear or be removed from `document.body` from which modal (if using multiple modals simultaneously).
+You can use the `className` and `overlayClassName` props to specify a given CSS
+class for each of those.
 
-Note: If you provide those props all default styles will not be applied, leaving all styles under control of the CSS class.
+You can override the default class that is added to `document.body` when the
+modal is open by defining a property `bodyOpenClassName`.
+
+It's required that `bodyOpenClassName` must be `constant string`, otherwise we
+would end up with a complex system to manage which class name should appear or
+be removed from `document.body` from which modal (if using multiple modals
+simultaneously).
+
+`bodyOpenClassName` can support adding multiple classes to `document.body` when
+the modal is open. Add as many class names as you desire, delineated by spaces.
```

Comment:
This section

> `bodyOpenClassName` can support adding multiple classes to `document.body` when
> the modal is open. Add as many class names as you desire, delineated by spaces.

is the only newly added documentation. The rest of the changes in this file were simply refactors of the existing documentation into a more easily editable form (e.g. line breaks at ~80 characters, newlines between separate paragraphs, [removal of duplicate spacing](https://github.com/reactjs/react-modal/pull/452/files#diff-80a7f6b1e8f2f0a7f4b86e5ebca280a4L3), etc.)

If you'd prefer I undo these changes and merely add my documentation, just let me know.